python2/raygun4py/raygunmsgs.py

```diff
@@ -173,6 +180,28 @@ def __init__(self, exc_type, exc_value, exc_traceback, options):
     def get_classname(self):
         return self.className
 
+    def _get_frames(self, exc_traceback):
+        if not exc_traceback or inspect.istraceback(exc_traceback):
+            return inspect.getinnerframes(exc_traceback)
+
+        if self._is_stack_frame_type(exc_traceback):
+            return exc_traceback
+
+        raise DeveloperException("Expected traceback type. Got '{}' instead.".format(type(exc_traceback)))
+
+    def _is_stack_frame_type(self, exc_traceback):
+        if type(exc_traceback) != self.INSPECT_STACK_BASE_TYPE:
+            return False
+
+        for frame in exc_traceback:
+            if type(frame) != self.INSPECT_STACK_CLASS_SUBTYPE:
+                return False
+            else:
+                if not inspect.isframe(frame[0]):
+                    raise DeveloperException("Expected frame type. Got '{}' instead.".format(type(frame[0])))
```

Comment:
I've seen other projects (outside of raygun) use this style. (`raise DeveloperException`) Not exactly sure if it's out of convention.

My goal is to relate when a developer may have made a mistake by passing an incorrect argument.

but seeing how this is Raygun.. would it be better to send this information to Raygun instead? Silence the python exception and send it to Raygun? Maybe `return None`, which would give the developer a blank traceback, in which case, maybe they'd realize they are using raygun package incorrectly.

```suggestion
                    return None
```